csharp/extractor/Semmle.Extraction.CSharp/Entities/TypeMention.cs

```diff
@@ -90,7 +90,11 @@ protected override void Populate(TextWriter trapFile)
                     var tts = (TupleTypeSyntax)syntax;
                     var tt = (TupleType)type;
                     Emit(trapFile, loc ?? syntax.GetLocation(), parent, type);
-                    tts.Elements.Zip(tt.TupleElements, (s, t) => Create(Context, s.Type, this, t.Type)).Enumerate();
+                    foreach (var (s, t) in tts.Elements.Zip(tt.TupleElements, (s, t) => (s, t?.Type)))
```

Comment:
Shouldn't we be more explicit, and use `GetTupleElementsMaybeNull()` here?

Reply:
Yes, definitely. I was sure I had done that, but I guess not.

Reply:
No, wait, that is actually the field `Field?[] TupleElements` on the class `TupleType`, which uses `GetTupleElementsMaybeNull` internally.